src/mon/MonmapMonitor.cc

```diff
@@ -173,8 +173,9 @@ bool MonmapMonitor::preprocess_command(MonOpRequestRef op)
 
   if (prefix == "mon stat") {
     mon->monmap->print_summary(ss);
-    ss << ", election epoch " << mon->get_epoch() << ", quorum " << mon->get_quorum()
-       << " " << mon->get_quorum_names();
+    ss << ", election epoch " << mon->get_epoch() << ", leader "
```

Comment:
the first one in quorum is the leader. so this information is already included in `mon->get_quorum_names()`, right?